code/clue.js

```diff
@@ -1,59 +1,454 @@
+  mysteryObject.forEach((object) => {
+    object.favourite = randomSelector(favouriteObject).id;
+  });
+};
+
+/* Toggles the loader animation */
+const toggleAnimation = (loaderId) => {
+  get(loaderId).classList.toggle('loader-container-animation');
+};
+
+/* Hides the loader image when a card is picked */
+const toggleLoader = (loaderId) => {
+  style(loaderId).display = 'none';
+};
+
+/* Hides the current window and displays the next */
+const toggleWindow = (windowId) => {
+  const windowElement = get(windowId);
+  toggleWindowClass(windowId);
+  toggleWindowClass(windowElement.nextElementSibling.id);
+};
+
+/* Toggles between classes to hide/display a window */
+const toggleWindowClass = (id) => {
+  get(id).classList.toggle('visible');
+  get(id).classList.toggle('invisible');
+};
+
+/* Hides the reveal the crime button when the game is started */
+const hideButton = () => {
+  const button = (style('revealButton').visibility = 'hidden');
+};
+
+/* Increases the chance of the referred obeject's favourite property to be picked */
+const increaseChance = (object) => {
+  if (object === mystery.killer) {
+    arrayDuplicated = duplicateArray(findFavourite(mystery.killer, weapons));
+    return randomFavourite(weapons, arrayDuplicated);
+  } else if (object === mystery.weapon) {
+    arrayDuplicated = duplicateArray(findFavourite(mystery.weapon, rooms));
+    return randomFavourite(rooms, arrayDuplicated);
+  }
+};
+
+/* The function is called to find the favourite object that matches the id of 
+the favourite property of the mystery object */
+const findFavourite = (mysteryObject, favouriteObjects) => {
+  const favourite = favouriteObjects.find(
+    (favourite) => favourite.id === mysteryObject.favourite
+  );
+  return favourite;
+};
+
+/* The function creates a new array where the favourite object occurs a random
+ number of times */
+const duplicateArray = (favourite) => {
+  const randomNumber = Math.floor(Math.random() * 11);
+  const arrayDuplicated = new Array(randomNumber).fill(favourite);
+  return arrayDuplicated;
+};
+
+/* The functions merges two arrays to create a new array where the favourite object occurs more than once */
+const randomFavourite = (favourite, arrayDuplicated) => {
+  const increasedFavourite = [...favourite, ...arrayDuplicated];
+  return randomSelector(increasedFavourite);
+};
+
+const showInfo = (loaderId) => {
+  if (
+    loaderId === 'loaderWeapon' ||
+    (loaderId === 'loaderRoom' && !killerPicked)
+  ) {
+    get('info').innerHTML = 'Start by picking a killer card';
+  } else if (loaderId === 'loaderRoom' && pickKiller) {
+    get('info').innerHTML = 'Go ahead, pick a weapon card';
+  }
+};
+
+/* Capitalizes the first letter */
+const capitalizeFirstLetter = (inputToCheck) => {
+  return inputToCheck[0].toUpperCase() + inputToCheck.slice(1);
+};
 
-const weapons = []
+/* Function that returns an object to which the id refers */
+const get = (id) => {
+  return document.getElementById(id);
+};
```

Comment:
Maybe a more descriptive function name here would be "getElement" or "getDocument"